packages/@okta/vuepress-site/docs/guides/custom-sms-messaging/main/index.md

```diff
@@ -4,21 +4,21 @@ excerpt: Learn how to customize the SMS messages sent to your users.
 layout: Guides
 ---
 
-This guide teaches you how to customize the text of the SMS messages that Okta sends to an end user for multifactor authentication verification.
+This guide explains how to customize the text of the SMS messages that Okta sends to an end user for multifactor authentication verification.
 
 ---
 
 **Learning outcomes**
 
-Customize SMS messages
+Customize SMS messages.
 
 **What you need**
 
-An [Okta Developer Edition organization](/signup)
+[Okta Developer Edition organization](https://developer.okta.com/signup)
 
 ---
 
-## Overview
+## About text customization of SMS messages
```

Comment:
SMS messages are always text, and they are messages, so this seems a bit redundant in places. How about "SMS customization overview"?

Reply:
👍